File: Samples/azure-storage/Azure.Java/models/StorageAccountKeys.java
/**
 */

package petstore.models;


/**
 * The access keys for the storage account.
 */
public class StorageAccountKeys {
    /**
     * Gets the value of key 1.
     */
    private String key1;

    /**
     * Gets the value of key 2.
     */
    private String key2;

    /**
     * Get the key1 value.
     *
     * @return the key1 value
     */
    public String key1() {
        return this.key1;
    }

    /**
     * Set the key1 value.
     *
     * @param key1 the key1 value to set
     * @return the StorageAccountKeys object itself.
     */
    public StorageAccountKeys withKey1(String key1) {
        this.key1 = key1;
        return this;
    }

    /**
     * Get the key2 value.
     *
     * @return the key2 value
     */
    public String key2() {
        return this.key2;
    }

    /**
     * Set the key2 value.
     *
     * @param key2 the key2 value to set
     * @return the StorageAccountKeys object itself.
     */
    public StorageAccountKeys withKey2(String key2) {
        this.key2 = key2;
        return this;
    }

}

File: Samples/azure-storage/Azure.CSharp/Models/StorageAccountKeys.cs

namespace Petstore.Models
{
    using System.Linq;

    /// <summary>
    /// The access keys for the storage account.
    /// </summary>
    public partial class StorageAccountKeys
    {
        /// <summary>
        /// Initializes a new instance of the StorageAccountKeys class.
        /// </summary>
        public StorageAccountKeys()
        {
          CustomInit();
        }

        /// <summary>
        /// Initializes a new instance of the StorageAccountKeys class.
        /// </summary>
        /// <param name="key1">Gets the value of key 1.</param>
        /// <param name="key2">Gets the value of key 2.</param>
        public StorageAccountKeys(string key1 = default(string), string key2 = default(string))
        {
            this.Key1 = key1;
            this.Key2 = key2;
            CustomInit();
        }

        /// <summary>
        /// An initialization method that performs custom operations like setting defaults
        /// </summary>
        partial void CustomInit();

        /// <summary>
        /// Gets the value of key 1.
        /// </summary>
        [Newtonsoft.Json.JsonProperty(PropertyName = "key1")]
        public string Key1 { get; set; }

        /// <summary>
        /// Gets the value of key 2.
        /// </summary>
        [Newtonsoft.Json.JsonProperty(PropertyName = "key2")]
        public string Key2 { get; set; }

    }
}
